mne/channels/tests/test_standard_montage.py

```diff
@@ -63,20 +64,9 @@ def test_standard_montages_on_sphere(kind, tol, head_size):
 
 def test_standard_superset():
     """Test some properties that should hold for superset montages."""
-    with pytest.deprecated_call():
-        o_1005 = read_montage('standard_1005')  # old montages
-        o_1020 = read_montage('standard_1020')
     # new montages, tweaked to end up at the same size as the others
     m_1005 = make_standard_montage('standard_1005', 0.0970)
     m_1020 = make_standard_montage('standard_1020', 0.0991)
     assert len(set(m_1005.ch_names) - set(m_1020.ch_names)) > 0
     # XXX weird that this is not a proper superset...
     assert set(m_1020.ch_names) - set(m_1005.ch_names) == {'O10', 'O9'}
-    c_1005 = m_1005._get_ch_pos()
-    for key, value in m_1020._get_ch_pos().items():
-        want = o_1020.pos[o_1020.ch_names.index(key)]
-        assert_allclose(value, want, err_msg=key, atol=1e-4)
-        if key not in ('O10', 'O9'):
-            assert_allclose(o_1005.pos[o_1005.ch_names.index(key)], want,
-                            err_msg=key, atol=1e-4)
-            assert_allclose(c_1005[key], value, atol=1e-4, err_msg=key)
```

Comment:
This line should not be removed (just remove the ones dealing with `o_*`). It makes sure that the 1005 for a given channel is the same as the 1020 pos.